server/src/test/java/org/opensearch/action/main/MainResponseTests.java

```diff
@@ -102,6 +102,36 @@ public void testToXContent() throws IOException {
           + "}", Strings.toString(builder));
     }
 
+    public void testToXContent_clientCompatibilityVersion() throws IOException {
+        String clusterUUID = randomAlphaOfLengthBetween(10, 20);
+        final Build current = Build.CURRENT;
+        Build build = new Build(
+            current.type(), current.hash(), current.date(), current.isSnapshot(),
+            current.getQualifiedVersion(), current.getDistribution()
+        );
+        Version version = Version.CURRENT;
+        final Version clientCompatibilityVersion = LegacyESVersion.V_7_10_2;
+        MainResponse response = new MainResponse("nodeName", version, new ClusterName("clusterName"), clusterUUID,
+            build, clientCompatibilityVersion);
+        XContentBuilder builder = XContentFactory.jsonBuilder();
+        response.toXContent(builder, ToXContent.EMPTY_PARAMS);
+        assertEquals("{"
```

Comment:
This test should only be focused on the field, otherwise it's hard to tell what we're after and it can break for unrelated changes.

Reply:
updated this additional test to only check for the version number.